src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/mgr-summary.pipe.ts

```diff
@@ -15,18 +15,22 @@ export class MgrSummaryPipe implements PipeTransform {
     }
 
     let activeCount = this.i18n('n/a');
-    const titleText = _.isUndefined(value.active_name)
+    const activetitleText = _.isUndefined(value.active_name)
```

Comment:
Please use camel-case.

```suggestion
    const activeTitleText = _.isUndefined(value.active_name)
```

Reply:
okay, will do that

Reply:
Made changes as suggested